build-system/tasks/presubmit-checks.js

```diff
@@ -174,7 +174,7 @@ var forbiddenTerms = {
   'cidServiceForDocForTesting': {
     message: privateServiceFactory,
     whitelist: [
-      'extensions/amp-analytics/0.1/cid-impl.js',
+      'src/service/cid-impl',
```

Comment:
Doh! Thanks, done and pushed. Got some errors in Travis that need manual checks and I think cid-impl needs to be added to a white list because of it's cidServiceForDocForTesting use?